match_against_all_templates/input_text_files/test.txt

```diff
@@ -0,0 +1,5 @@
+Copyright (C) 2006 by Rob Landley <rob@landley.net>
```

Comment:
Why do we need these test.txt files in every folder? Are they being used somewhere?

Reply:
These are just sample files used as examples for checking the CLI of the features. I am thinking that would help users to use this as helper examples but if you think they are not needed I will remove them when everything gets finalised.

Reply:
Keep these either in example folder or resources folder. Example folder will contain the code for using the tool by importing it. (Will help user more)